options/ansi/generic/signal-stubs.cpp

```diff
@@ -7,13 +7,20 @@
 #include <mlibc/sysdeps.hpp>
 
 __sighandler signal(int sn, __sighandler handler) {
-	mlibc::infoLogger() << "\e[31mmlibc: signal() always returns SIG_DFL\e[39m" << frg::endlog;
 	if(!mlibc::sys_sigaction) {
 		MLIBC_MISSING_SYSDEP();
 		errno = ENOSYS;
 		return SIG_ERR;
 	}
 	if(handler == SIG_DFL || handler == SIG_IGN) {
```

Comment:
Do we need this special case? Doesn't it to the same as the case below?

Reply:
It does something different, in this case we return the old handler, in the other case we don't have to do that